app/controllers/admin/edition_workflow_controller.rb

```diff
@@ -167,7 +171,7 @@ def approve_retrospectively
 private
 
   def get_layout
-    design_system_actions = %w[confirm_approve_retrospectively confirm_force_schedule confirm_unpublish confirm_unschedule confirm_unwithdraw unpublish confirm_force_publish]
+    design_system_actions = %w[confirm_approve_retrospectively confirm_force_schedule confirm_publish confirm_unpublish confirm_unschedule confirm_unwithdraw unpublish confirm_force_publish]
```

Comment:
At this point do we still need this? They're all using the design system now, right?

Reply:
Ooh I think so, I'll have a go at removing it in a separate PR just to keep it separate and hopefully clearer